hack/validate-release.sh

```diff
@@ -4,6 +4,7 @@ set -e
 set -u
 
 SUBSCRIPTION="./test/operator-hub/subscription2.yaml"
+BUNDLE_VERSION=`curl -s https://raw.githubusercontent.com/operator-framework/community-operators/master/community-operators/service-binding-operator/service-binding-operator.package.yaml|grep "currentCSV" | cut -d "v" -f 3`
```

Comment:
IMHO, parsing the value here is pretty fragile, especially using `v` char as a delimiter.

Let's use `yq` or similar tool to get the latest version available on an a given channel, something like this:

```bash
curl -s https://raw.githubusercontent.com/operator-framework/community-operators/master/community-operators/service-binding-operator/service-binding-operator.package.yaml 
| yq -r '.channels[] | select (.name == "alpha") | .currentCSV | sub("service-binding-operator\\.v"; "") ' 
```

I would make the channel name parametrizable as well.

Reply:
I have used `yq` now. 46fa128f143229bbd8e207ee3b8ea97ca33d46e2

Reply:
As mentioned above, please make possible to pass channel name as argument to `validate-release.sh` script.